modules/gapi/include/opencv2/gapi/gopaque.hpp

```diff
@@ -334,6 +334,9 @@ template<typename T> class GOpaque
     detail::GOpaqueU m_ref;
 };
 
+// NB: Used by python wrapper
+using GSize = GOpaque<cv::Size>;
+using GRect = GOpaque<cv::Rect>;
```

Comment:
Does it make sense to implement wrapper for GArray & GOpaque to avoid making using for every GArray & GOpaque type ?

Reply:
Same

Reply:
Fixed